src/color-swatch/color-swatch.js

```diff
@@ -143,6 +165,28 @@ const Self = class ColorSwatch extends NudeElement {
 			},
 			reflect: false,
 		},
+		coords: {
+			type: {
+				is: Array,
+				values: {
+					is: Object,
+					defaultKey: (value, i) => {
+						let [space, channel] = value.split(".");
+						if (!channel) {
+							channel = space;
+							space = this.color?.space.id;
+						}
+
+						return Color.Space.get(space)?.coords[channel]?.name ?? channel;
```

Comment:
Please use `Color.Space.resolveCoord()` which exists exactly for this purpose: https://github.com/color-js/color.js/blob/main/src/ColorSpace.js#L370

Reply:
With great pleasure! TIL. Thanks for sharing it with me.